cardano-explorer/src/Explorer/Web/ClientTypes.hs

```diff
@@ -116,7 +116,7 @@ adaToCCoin (Ada ada) = CCoin $ T.pack $ showFixed True $ ada * 1000000
 data CBlockEntry = CBlockEntry
     { cbeEpoch      :: !Word64
     , cbeSlot       :: !Word16
-    , cbeBlkHeight  :: !Word
+    , cbeBlkHeight  :: !(Maybe Word)
```

Comment:
changing the type in here will break the compatibility with the old explorer clients, would be better to use `maybe` or `fromMaybe` to convert it down

the api should return a `Left` for EBB's, so they never have to be represented by a `CBlockEntry`

Reply:
Yeah, I can use a `fromMaybe`